app/src/main/java/org/oppia/app/player/state/itemviewmodel/InteractionViewModelModule.kt

```diff
@@ -84,15 +84,4 @@ class InteractionViewModelModule {
       )
     }
   }
-
-  @Provides
-  @IntoMap
-  @StringKey("DragAndDropSortInput")
```

Comment:
Why isn't this needed anymore?

Reply:
this is there in the codebae

Reply:
might have been a duplicate because of stacked PR